client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -111,37 +109,12 @@ export const receiveCommentSuccess = ( store, action, response ) => {
 	} );
 };
 
-export const receiveCommentError = ( { dispatch, getState }, { siteId, commentId } ) => {
-	const site = getReaderSite( getState(), siteId );
-	const siteName = getReaderSiteName( { site } );
-
-	if ( siteName ) {
-		dispatch(
-			errorNotice(
-				translate( 'Failed to retrieve comment for site “%(siteName)s”', {
-					args: { siteName },
-				} ),
-				{ id: `request-comment-error-${ siteId }` }
-			)
-		);
-	} else {
-		const rawSite = getRawSite( getState(), siteId );
-		const error =
-			rawSite && rawSite.name
-				? translate( 'Failed to retrieve comment for site “%(siteName)s”', {
-						args: { siteName: rawSite.name },
-					} )
-				: translate( 'Failed to retrieve comment for your site' );
-
-		dispatch( errorNotice( error, { id: `request-comment-error-${ siteId }` } ) );
-	}
-
+export const receiveCommentError = ( { dispatch, getState }, { siteId, commentId } ) =>
```

Comment:
no need to destructure `getState`. (why the linter didn't catched it?)

Reply:
Heyyyyy, great catch – yeah, I don't get any linter complaints either.